sklearn/manifold/t_sne.py

```diff
@@ -385,12 +385,12 @@ def trustworthiness(X, X_embedded, n_neighbors=5, precomputed=False):
     .. math::
 
         T(k) = 1 - \frac{2}{nk (2n - 3k - 1)} \sum^n_{i=1}
-            \sum_{j \in U^{(k)}_i} (r(i, j) - k)
+            \sum_{j} \max(0, (r(i, j) - k))
 
-    where :math:`r(i, j)` is the rank of the embedded datapoint j
-    according to the pairwise distances between the embedded datapoints,
-    :math:`U^{(k)}_i` is the set of points that are in the k nearest
-    neighbors in the embedded space but not in the original space.
+    where for each sample i, j is among its k nearest neighbors in the output
```

Comment:
output -> embedded?

Reply:
I made this change according to @tomMoral comment:
> Maybe use the "output space" would be clearer? input/output seem clearer when talking about mappings.

Both ways seem clear to me: "embedded" is more precise, but maybe "output" is clear enough in this case and maybe simpler ?

Reply:
I'm not too fussed either way

Reply:
I'm not too fussed either way